embedded-graphics/src/mock_display.rs

```diff
@@ -207,11 +247,46 @@ where
 
         mirrored
     }
+
+    /// Maps a `MockDisplay<C>' to a `MockDisplay<CT>` by applying a function
+    /// to each pixel.
+    ///
+    /// # Examples
```

Comment:
It took me a bit long than I'd like to admit to figure out what this example does, so a heading would help clarify it at a glance IMO.

```suggestion
    /// # Examples
    ///
    /// ## Invert the colors in a `BinaryColor` pattern
```

Reply:
Good point! I've added a comment.